server/src/middlewares/auth.middleware.ts

```diff
@@ -0,0 +1,30 @@
+import { NextFunction, Response } from 'express';
+import { TokenExpiredError } from 'jsonwebtoken';
+import AuthError, { AuthErrorType } from '../errors/auth.error';
+import { IAuthRequest } from '../interfaces/authRequest.interface';
+import authService from '../services/auth.service';
+
+const authMiddleware = async (req: IAuthRequest, res: Response, next: NextFunction): Promise<void> => {
+  const { authorization } = req.headers;
+
+  if (authorization === null || authorization === undefined) {
```

Comment:
```suggestion
  if (!authorization) {
```
한번에 표현할 수 있을 것 같아요! ☺️

Reply:
저는 명시해주는걸 선호하는 편입니다 ㅎㅎ